jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/security/SecurityUtils.java

```diff
@@ -142,9 +130,11 @@ public static void checkPermission(Permission permission) throws SecurityExcepti
      */
     public static <T> T doPrivileged(PrivilegedAction<T> action)
     {
-        if (!USE_SECURITY_MANAGER || doPrivileged == null)
+        // Keep this method short and inlineable.
+        MethodHandle methodHandle = doPrivileged;
```

Comment:
not sure why the local `methodHandle` variable is needed?  `doPrivileged` is final.

Reply:
I'm not sure either, it's in 12.0 where I copied this from.
Perhaps its required for Java to automatically inline the function ? I don't know much about the automatic inlining behaviour.

I have removed it.